### DISCUSSION TITLE
Net8, I need to create a Blazor Webassemby app with Blazorise and bootstrap5

### DISCUSSION THREAD

**Discussion by domenicocanavero** [Megabit/Blazorise#5632] [Q&A]
I used the blazorise template 1.5, Visual Studio 2022 and .Net 8.
I create a new project using Blazorise App (Megabit) template, Net8, BlazorHost: Blazor Web Assembly, Provider: bootstrap5.
When I run the project:
                                - with debug, error message: Process with an Id of 29324 is not running.
                                - without, unable to connect to web server 'Project.Client'. The web server is no longer running.

The template creates only one project, the server side and not the client side project.
Attached the solution.

I need to create a Blazor Webassembly app with Blazorise/bootstrap and Net8, which is the correct procedure?
Thanks

[Project1.zip](https://github.com/user-attachments/files/16349100/Project1.zip)

**Comment by stsrki** [comment]
@David-Moreira can you help?

**Comment by David-Moreira** [comment]
I just tested 
- Your zip, works fine.
- Creating a new app with the settings you mention, works fine.
![image](https://github.com/user-attachments/assets/1d50e062-820e-454a-a796-439302e63eed)

That error seems to be from something else. Does a regular Blazor WebAssembly not yield the same error on your environment?

**Reply by David-Moreira** [reply]
Also, try making sure IIS express is not left running from running something else, or just restart your computer so you're sure you have a clean slate.

**Reply by David-Moreira** [reply]
also, here's a quick google search for your error:

https://stackoverflow.com/questions/26424902/process-with-an-id-of-is-not-running-in-visual-studio

![image](https://github.com/user-attachments/assets/15e780e1-686d-4563-9f5f-e13e891b43bc)

**Comment by domenicocanavero** [comment]
You are right, I have tryed with my laptop and the project runs, so is a problem of my PC, thanks. 
I'm used the standard VS 2022 template, Blazor Web App - Interactive mode WebAssemply, it creates 2 projects: host and client and by default runs the host project.
With the Blazorise template only one project and it runs project.client that not exist ... this has confused me a bit ... anyway, thanks again.
